src/ensembl/production/workflow/hive.py

```diff
@@ -1,12 +1,16 @@
 def construct_pipeline(job, spec):
     "construct cmd for pipeline to run"
-    hive_dbname = spec['user'] + '_' + job['PipelineName'] + '_' + str(spec['ENS_VERSION']) + '_QRP'
+    
+    hive_dbname = spec['user'] + '_' + job['PipelineName'] + '_' + str(spec['ENS_VERSION']) 
+    queue_name = 'production-rh74' if job.get('HOST', None) == 'NOAH' else 'debug'
```

Comment:
No need for that, consider that either everything is run from same host. `OR` we could consider having the target "run / config" host set up in config files instead.